src/virtual_dom/node/el.rs

```diff
@@ -106,8 +106,15 @@ impl<Ms> El<Ms> {
 
     // todo: Return El instead of Node here? (Same with from_html)
     /// Create elements from a markdown string.
+    /// Enables all additional markdown extensions, see #392
     pub fn from_markdown(markdown: &str) -> Vec<Node<Ms>> {
-        let parser = pulldown_cmark::Parser::new(markdown);
+        let mut options = pulldown_cmark::Options::empty();
+        options.insert(pulldown_cmark::Options::ENABLE_TABLES);
+        options.insert(pulldown_cmark::Options::ENABLE_FOOTNOTES);
+        options.insert(pulldown_cmark::Options::ENABLE_STRIKETHROUGH);
+        options.insert(pulldown_cmark::Options::ENABLE_TASKLISTS);
```

Comment:
Can we use https://docs.rs/pulldown-cmark/0.7.0/pulldown_cmark/struct.Options.html#method.all?

Reply:
Yes, I misunderstood the documentation for that method.

Reply:
Yeah I was also fighting with the docs and it looks like it's created in the macro so there isn't even code for it. Just try it, I hope it'll work.